docs/user_guides/fs/compute_engines.md

```diff
@@ -8,7 +8,8 @@ As such, Hopsworks supports three computational engines:
 
 1. [Apache Spark](https://spark.apache.org): Spark Dataframes and Spark Structured Streaming Dataframes are supported, both from Python environments (PySpark) and from Scala environments.
 2. [Pandas](https://pandas.pydata.org/): For pure Python environments without dependencies on Spark, Hopsworks supports [Pandas Dataframes](https://pandas.pydata.org/).
-3. [Apache Flink](https://flink.apache.org) *experimental*: Flink Datastreams are currently supported as an experimental feature, you can reach out to Hopsworks for guidance.
+3. [Apache Flink](https://flink.apache.org) *experimental*: Flink Data Streams are currently supported as an experimental feature from Java/Scala environments.
```

Comment:
I would remove the `experimental` for Flink - we have good documentation, examples and code.